abstract_specification.md

```diff
@@ -1131,17 +1111,6 @@ The format and contents of a Context object is completely implementation-defined
 An implementation may provide a way to construct its specific context type, which can be used by user setup code to connect to the SOMA data store (represented above as the `somaimpl.create_context` call).
 However, client code should treat the `context` object on any instantiated SOMA objects as an opaque value, and only pass it directly as the context parameter when creating or opening other SOMA data.
 
-# ⚠️ Other Issues (open issues with this doc)
-
-Issues to be resolved:
-
-1. Are there operations specific to SOMAExperiment and SOMAMeasurement that need to be defined? Or do they inherit only the ops from SOMACollection?
-2. What (if any) additional semantics around writes need to be defined?
```

Comment:
Deleting #2 for good. I believe write semantics have been well documented now in the `Object Lifecycle` section, w.r.t. atomicity and concurrency semantics.